config/routes.rb

```diff
@@ -23,7 +23,7 @@
 
   # resource :global_stats, only: [:show], defaults: {format: :json}
 
-  match "/pixels/:pixel_id/convert/:impression_id", to: "pixel_conversions#create", via: [:get, :post], as: "pixel_conversions"
+  match "/pixels/:pixel_id/impressions/:impression_id", to: "pixel_conversions#create", via: [:get, :post], as: "pixel_conversions"
```

Comment:
Nate used impressions in the XHR request which was different from the route specified but it made more sense to me to have the route structured this way so I modified the route instead of the fetch.